src/concurrent/mod.rs

```diff
@@ -167,7 +438,18 @@ pub struct ProjectMetrics {
     pub average: Metrics,
 }
 
-#[derive(Default)]
+impl ProjectMetrics {
+    const fn new(total: Metrics, min: Metrics, max: Metrics, average: Metrics) -> Self {
+        Self {
+            total,
+            min,
+            max,
+            average,
+        }
+    }
+}
+
+#[derive(Debug, Default, Clone, Copy)]
```

Comment:
I derived `Copy` for this structure so that I don't have to use a reference (all fields being `f64`) when it is passed to other functions, for example when it is passed to `MetricsData::file(project_data, ...)`.